src/main/java/edu/mbrooks/advancedjava/impl/Person.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Program Name:    Person.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the person interface
+ */
+package edu.mbrooks.advancedjava.impl;
```

Comment:
impl definitely should not be a package name. Interfaces and Implementations should typically exist in the same package.

Reply:
This is the coding standard I've seen where I work... I'll keep this in mind for the future - also answers the question I posted with the assignment